.ci/.docker-images.schema.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
❤️

Reply:
👍 This is a good approach. I would like to see us do much more of this.

Reply:
Recently I found https://github.com/bitnami-labs/readme-generator-for-helm, which makes generating docs and OpenAPI schema files so easy that there is no reason not to do it.

values.yaml
```yaml
## @section opbeans Opbeans Ruby Helm chart configuration
opbeans:
  ## @param opbeans.enabled Enable or disable the whole kubernetes deployments, this allow to use it on parent charts.
  enabled: true
  ## @param opbeans.nameOverride Overrides the the Helm Chart name used to naming kubernetes objects, by default `opbeans`.
  nameOverride: ""
  ## @param opbeans.fullnameOverride Overrides the the Helm Chart fullname used to naming kubernetes objects, by default `opbeans`.
  fullnameOverride: ""
  ## @section opbeans.image Docker image settings
  image:
    ## @param opbeans.image.name Name of the Docker image (docker.elastic.co/observability-ci/it_opbeans-frontend_nginx).
    ## @param opbeans.image.tag Tag of the Docker image (latest).
    ## @param opbeans.image.pullPolicy Docker image pull policy (IfNotPresent|Always|Never)
    name: docker.elastic.co/observability-ci/opbeans-ruby
    tag: daily
    pullPolicy: IfNotPresent
```

```bash
git clone https://github.com/bitnami-labs/readme-generator-for-helm; 
cd readme-generator-for-helm; 
git checkout 2.4.3 -b 2.4.3
npm install ./readme-generator-for-helm; 
echo "## Parameters" > ./README.md 
./node_modules/.bin/readme-generator --values ./values.yaml --readme ./README.md --schema ./values.schema.json
```